### DISCUSSION TITLE
How to change favicon ?

### DISCUSSION THREAD

**Discussion by andre0ani** [chamilo/chamilo-lms#5170] [General]
Hi,

I search, test, replace, but I can't put my own favicon, with the Academica theme.
Any suggestions please ?

Cordially,
andre0ani

**Comment by Kaneda-1** [comment]
Hello @andre0ani 
Locate the favicon.ico file on your site and replace it.
You must clear the cache from the administration for the changes to be displayed.

> Cleanup of cache and temporary files
> .../main/admin/archive_cleanup.php

**Comment by andre0ani** [comment]
Thanks @Kaneda-1 
It was just a cache problem...
